docs/integrations/fetching-incidents.md

```diff
@@ -93,11 +93,11 @@ incident = {
 
 ### Setting Last Run
 When the last of the events have been retrieved, we need to save the new last run time to the integration context. This timestamp will be used the next time the ```fetch-incidents``` function is run.
-When setting the last run object, it's important to know that the values of the dictionairy must be of type `string`.
+When setting the last run object, it's important to know that the values of the dictionary must be of type `string`.
 
 ```python
 demisto.setLastRun({
-  'start_time': datetime.now()
+  'start_time': datetime.now().isoformat()
```

Comment:
i think it is a bit confusing because we don't really recommend to store the now time as the last run, but the last fetched event timestamp

Reply:
so what are you suggesting? something like:
```
  'start_time': timestamp_to_datestring(last_incident['time'])
```

Reply:
we can do something like that, but maybe add a comment to explain cause its not clear whats `last_incident`